libultraship/libultraship/DummyController.cpp

```diff
@@ -0,0 +1,61 @@
+#include "DummyController.h"
+
+namespace Ship {
+	DummyController::DummyController(const std::string& CUID, const std::string& KeyName, bool Connected) {
+		GUID = CUID;
+		isConnected = Connected;
+		ButtonName = KeyName;
+	}
+
+	void DummyController::ReadFromSource(int32_t virtualSlot) {
+		
+	}
+
+	const std::string DummyController::GetControllerName() {
+		return GUID;
+	}
+
+	const std::string DummyController::GetButtonName(int32_t virtualSlot, int32_t n64Button) {
+		return ButtonName;
+	}
+
+	void DummyController::WriteToSource(int32_t virtualSlot, ControllerCallback* controller){
+		
+	}
+
+	bool DummyController::Connected() const {
+		return isConnected;
+	}
+
+	bool DummyController::CanRumble() const {
+		return false;
+	}
+
+	bool DummyController::CanGyro() const {
+		return false;
+	}
+
+	void DummyController::CreateDefaultBinding(int32_t slot) {
+		
+	}
+
+	std::string DummyController::GetControllerType() {
+		return "Unk";
```

Comment:
is there a reason this is "Unk" specifically? i feel like i'd have gone for "Undefined" or something

Reply:
CC @KiritoDv 

Disconnected may have been a better name. Let's make a note to revisit this later.